lib/administrate/field/deferred.rb

```diff
@@ -29,8 +29,8 @@ def searchable?
         options.fetch(:searchable, deferred_class.searchable?)
       end
 
-      def searchable_field
-        options.fetch(:searchable_field)
+      def searchable_fields
+        options.fetch(:searchable_fields)
```

Comment:
This will end up being a breaking change. To reduce the impact of breakages, what do you think about still keeping `searchable_field` method, but in a way that reformats it for the new method?

We could then use [`ActiveSupport::Deprecation`](https://api.rubyonrails.org/classes/ActiveSupport/Deprecation.html) to highlight it and remove it in the release afterwards.

Reply:
Updated